src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkflowComponent.scala

```diff
@@ -216,13 +228,17 @@ trait WorkflowComponent {
         case None =>
```

Comment:
Can remove this case because flatMap takes partial functions and the `None` will continue